packages/lib-utils/src/app/redux/reducers/index.ts

```diff
@@ -1,5 +1,5 @@
 import { coreReducer } from './core';
-import { sdkK8sReducer } from './k8s';
+import sdkK8sReducers from './k8s';
```

Comment:
```suggestion
import sdkK8sReducer from './k8s';
```

It's only 1 reducer... right? You're assigning it as such on line 12.